lib/testing/src/lib/core/pages/info-drawer.page.ts

```diff
@@ -0,0 +1,75 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { element, by, ElementFinder } from 'protractor';
+import { BrowserVisibility } from './../utils/browser-visibility';
+import { TabsPage } from './material/tabs.page';
+
+export class InfoDrawerPage {
```

Comment:
I don't see it exported and used

Reply:
Thanks for the comment. I want to use it in ADW tests, so i should have exported it.

Reply:
It seems it was exported, check public-api.ts

Reply:
sorry I missed it

Reply:
No problem